addStudy.py

```diff
@@ -0,0 +1,101 @@
+# addStudy.py
+
+#!/usr/bin/python3
+
+# python addStudy.py -c rwa_configuration.yml -l
+# python addStudy.py -c rwa_configuration.yml -a "studyName"
+
+# addStudy.py
+#
+# Adding new studies to the database
+
+# Import our libraries
+import os
+import sys
+import argparse
+import yaml
+import psycopg2
+from include.calibrationLib import *
+
+# List all studies
+def get_studies(configuration, studName):
+    # return "List to populate"
+    sql = '''SELECT * FROM STUDY'''
+
+    return select(configuration["connection_string"], sql, {'name': None})
+
+# add studies
+def add_file(configuration, Name):
+    Name = str(Name)
+
+    sql = '''INSERT INTO study(Name) VALUES(%s) RETURNING id;'''
+
+    return select(configuration["connection_string"], sql, {'Name': Name})
+
+# connection
+def select(connectionString, sql, parameters):
+    # Open the connection
+    connection = psycopg2.connect(connectionString)
+
+    cursor = connection.cursor()
+    # add study
+    if (sql.find("INSERT") == 0):
+        cursor.execute(sql, (parameters['Name'],))
+        study_id = cursor.fetchone()[0]
+
+        connection.commit()
+
+        cursor.close()
+
+        return study_id
+    # list all studies
+    if (sql.find("SELECT") == 0):
+        cursor.execute(sql)
+
+        rows = cursor.fetchall()
+        for row in rows:
+            print(row)
+            # print(sql.find("SELECT"))
+        cursor.close()
```

Comment:
This entire function is unnecessary since `include/database.py` already has a `select` function. You should add new function to that library (e.g., `insert_returning`) and use the existing `select` function as needed.

Reply:
I am not sure if I am getting it right, please correct me if I am wrong here. You mean to say I can make changes in `database.py` and implement this functionality in existing `select` function.

Reply:
You can make changes to `database.py` but you shouldn't have to modify the `select` function. Remember that the function name should reflect what the function is doing (i.e., self-documenting code), so you will need to write a new function for the insert operation. Since you are doing an insert and return, the new function should be something along the lines of `insert_returning`.

Reply:
Okay. Thanks. 
Shall I open pull request for `database.py` first, so that you can review the exception handling. 

Later I can reopen this pull request and add changes in `database.py`(adding new function), and `addStudy.py` at the same time.

Reply:
That seems reasonable.